osu.Game/Graphics/OsuFont.cs

```diff
@@ -72,25 +75,17 @@ public static string GetFamilyString(Typeface typeface)
         /// <summary>
         /// Retrieves the string representation of a <see cref="FontWeight"/>.
         /// </summary>
-        /// <param name="typeface">The <see cref="Typeface"/>.</param>
-        /// <param name="weight">The <see cref="FontWeight"/>.</param>
-        /// <returns>The string representation of <paramref name="weight"/> in the specified <paramref name="typeface"/>.</returns>
-        public static string GetWeightString(Typeface typeface, FontWeight weight)
+        /// <param name="family">The font family.</param>
+        /// <param name="weight">The font weight.</param>
+        /// <returns>The string representation of <paramref name="weight"/> in the specified <paramref name="family"/>.</returns>
+        public static string GetWeightString(string family, FontWeight weight)
         {
-            if (typeface == Typeface.Torus && weight == FontWeight.Medium)
+            if ((family == @"Torus" || family == @"Torus-Alternate") && weight == FontWeight.Medium)
```

Comment:
Would have liked these to go via `family == GetFamilyString(Torus)`, etc...

Reply:
My bad, you can go ahead and apply in another PR to avoid this unnecessary duplication, or I’ll do it myself later.